collect_app/src/main/java/org/odk/collect/android/tasks/GoogleSheetsAbstractUploader.java

```diff
@@ -240,20 +240,19 @@ private boolean uploadOneSubmission(String id, String instanceFilePath, String j
         try {
             processInstanceXML(instanceFile, answersToUpload, mediaToUpload);
         } catch (FormException e) {
-            Timber.e(e, Collect.getInstance().getString(R.string.google_repeat_error));
```

Comment:
Note: this just says repeats aren't supported by the GDrive protocol. We don't need to log that.